lang/LangSource/PyrLexer.cpp

```diff
@@ -2111,7 +2110,9 @@ SCLANG_DLLEXPORT_C bool compileLibrary(bool standalone)
 	gNumCompiledFiles = 0;
 	compiledOK = false;
 
-	SC_LanguageConfig::readLibraryConfig(standalone);
+	if (!gLanguageConfig) {
+		SC_LanguageConfig::readLibraryConfig(standalone);
+	}
```

Comment:
With this change we don't read the config file from disk and instead use the in memory version